face_track/face_track.py

```diff
@@ -144,7 +144,8 @@ def __init__(self):
 
 		# Which face to look at
 		# rospy.Subscriber(self.TOPIC_FACE_TARGET, xxxFaceEvent, xxxself.face_event_cb)
-
+		rospy.Subscriber("/robot/perceived_text", String,
```

Comment:
I'm not sure, you may have to ask Vytas, but the topic to subscribe to should probably be `perceived_text` instead of `/robot/perceived_text`  -- the reason for this is that the locations of the topics are controlled by config files, which vary from one robot to another, and so you don't want to hard-code the paths into your code.